tests/utils/exec/rosacli/cmd_runner.go

```diff
@@ -35,6 +37,7 @@ func NewRunner() *runner {
 			debug:  false,
 			color:  "auto",
 		},
+		envs: os.Environ(),
```

Comment:
do we really need that ?

Reply:
In my testing when I would add an environment variable, then the runner would lose access to all the other environment variables which at least on my system would cause issues (e.g. with finding where the `rosa` binary is). It seems like the the behavior is when you set `cmd.Env` to `nil`, then it uses your system environment variables, but once you set `cmd.Env` to a value then it will only set those environment variables and you lose the system ones.

So I think we do need pass in all the other environment variables, but if you think that's better to do elsewhere I can move it